python/paddle/fluid/dygraph/nn.py

```diff
@@ -119,6 +119,10 @@ class Conv2D(layers.Layer):
         act (str): Activation type, if it is set to None, activation is not appended.
             Default: None
 
+    Parameters:
+        filter (Parameter): the learnable weights of filters of this layer.
```

Comment:
looks like it should not be call as Parameters, call them Attributes or Members maybe better